src/pages/caseDetails/FileView/FileTableView.jsx

```diff
@@ -1,20 +1,26 @@
-import React, { useContext } from "react";
-import { Container, withStyles } from "@material-ui/core";
+import React, { useContext } from 'react';
+import { Container } from '@material-ui/core';
 import {
   TableContext,
   TableView,
-  Wrapper,
+  btnTypes,
   onRowSeclect,
-} from "../../../bento-core";
-import { fileTable, fileWrapperConfig } from "../../../bento/caseDetailsData";
-// import { customTheme, themeConfig } from './Theme';
-import { CustomizeCellView } from "../../../components/PaginatedTable/Customize/CellView";
-import { ExtendedViewConfig } from "../../../components/PaginatedTable/Customize/ExtendedView";
-import { updateWrapperConfig } from "../../../components/PaginatedTable/Customize/TableView";
-import { themeConfig, customTheme } from "../SampleView/Theme";
+} from '../../../bento-core';
+import {
+  associateFilesTooltipConfig,
+  fileTable,
+} from '../../../bento/caseDetailsData';
+import { CustomizeCellView } from '../../../components/PaginatedTable/Customize/CellView';
+import { ExtendedViewConfig } from '../../../components/PaginatedTable/Customize/ExtendedView';
+import { themeConfig } from '../SampleView/Theme';
+import { AddSelectedFilesButton } from '../../../components/PaginatedTable/components/AddFilesButtons';
+import ViewJBrowseButton from '../../JbrowseDetail/components/JBrowseViewBtn';
+import * as Styled from './FilteTable.styled';
```

Comment:
Let us avoid this pattern `import * from <>`. We can import the ones we used. In the interest of bundle size and so webpack can handle thatt